lisa/testsuite.py

```diff
@@ -285,6 +293,64 @@ def simple_requirement(
     )
 
 
+def node_requirement(
+    node: Optional[schema.NodeSpace] = None,
+    supported_platform_type: Optional[List[str]] = None,
+    unsupported_platform_type: Optional[List[str]] = None,
+    supported_os: Optional[List[Type[OperatingSystem]]] = None,
+    unsupported_os: Optional[List[Type[OperatingSystem]]] = None,
+    environment_status: EnvironmentStatus = EnvironmentStatus.Connected,
+) -> TestCaseRequirement:
+    node = _construct_node(node)
```

Comment:
in this method, the node should be a required argument, and not need to construct. It's default value will be taken.